src/huggingface_hub/inference/_client.py

```diff
@@ -427,6 +463,7 @@ def automatic_speech_recognition(
         audio: ContentT,
         *,
         model: Optional[str] = None,
+        parameters: Optional[Dict[str, Any]] = None,
```

Comment:
I'm a bit chilly about adding a generic `parameters`. I feel that the goal of our client with several providers is to harmonize the experience. Adding `parameters` here means the user will have to know which fields to provide. I think it's fine if for now we don't provide any parameters and then do a `hf.js/tasks` + `huggingface_hub` update specific to automatic-speech-recognition

Reply:
Yes that was my understanding. But I'd prefer not to support any parameter rather than adding a generic one that will be more difficult to remove afterward.